lib/internal/loader/Loader.js

```diff
@@ -72,14 +100,20 @@ class Loader {
     return { url, format };
   }
 
+  // May create a new ModuleJob instance if one did not already exist.
   async getModuleJob(specifier, parentURL = this.base) {
```

Comment:
Your comment makes me wonder if this would be better called `getOrCreateModuleJob`.

Reply:
I think this is fine, `get` kind of leaves the details open of how the returned module job got into existence :)